cmd/list_test.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"testing"
+
+	fn "knative.dev/func"
+	"knative.dev/func/mock"
+)
+
+// TestList_Namespace ensures that list command options for specifying a
+// namespace (--namespace) or all namespaces (--all-namespacs) are  properly
+// evaluated.
+func TestList_Namespace(t *testing.T) {
+	_ = fromTempDirectory(t)
```

Comment:
```suggestion
func TestList_Namespace(t *testing.T) {
	t.Setenv("KUBECONFIG", "/tmp/non-existent")
	_ = fromTempDirectory(t)
```

Reply:
In order to not interfere with developers config.

Reply:
Ignoring KUBECONFIG of current user for unit tests now being tracked in a separate PR

Reply:
What PR @lkingland ? Has it been fixed?